src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -27,6 +27,11 @@ const REQUIRED_FORM_FIELDS = [
     CONTACT1_LASTNAME_INFO.fieldApiName
 ];
 
+const EXCLUDED_OBJECT_MAPPINGS = [
+    'Opportunity_Contact_Role_1_',
```

Comment:
This is a minor detail, but could you leave off the trailing underscore here and change the math on the substring below to be 'length - 10'? It both looks nicer, and would match to one of our default, protected mappings if for whatever reason we ever had those in play